vars/skipStage.groovy

```diff
@@ -15,7 +15,8 @@ String skip_stage_pragma(String stage, String def_val='false') {
 }
 
 boolean skip_ftest(String distro) {
-    return distro == 'ubuntu20' ||
+    return env['CI_FUNCTIONAL_' + distro + '_TEST'] == 'false' ||
```

Comment:
Parameters are also available as `params.<parameter-name>` such as is done here https://github.com/daos-stack/pipeline-lib/blob/faf1eb64ed7e8862bdaffe483ca0c1302bf401c6/vars/parseStageInfo.groovy#L166

I actually didn't know parameters were implicitly set into the environment.  I'm wondering about the pros/cons of using the environment hook for them vs the `params` object with regard to transparency about where they are actually coming from, etc.  I'm not sure I have a strong opinion, but `params` does seem more direct.

Reply:
Changing to the params object for next push.